tests/scripts/generate_psa_wrappers_future.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""Generate wrapper functions for PSA function calls.
+"""
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+### NOTE. This is a future version of the file located at
```

Comment:
Please don't duplicate code between the two repositories. It might work for work on the simulator, but it blocks all other work that would use the PSA wrappers, and making the PSA wrappers available to other work is the whole point of separating it out.

Reply:
Is there any current work that needs to modify the PSA wrappers? If not, this approach should be okay, with a piece at the end of the quarter to reconcile the code in the repositories. Otherwise the work on the framework repo is going to slow this down significantly

Reply:
This has been addressed in PR [#25 ](https://github.com/Mbed-TLS/mbedtls-framework/pull/25)